### DISCUSSION TITLE
Just stumbled onto this project and have been using it to better organize my rom library, and wanted to say fantastic job on this!

### DISCUSSION THREAD

**Discussion by ndguardian** [emmercm/igir#1083] [General]
I've been needing to do a bit of cleanup on my ROM archives, filtering out the ones with the languages I want, or building out custom ROM sets for different handheld units I've got. I am extremely please with the results I've been seeing with Igir.

Right now I use a hybrid workflow where RomVault to do my initial sorting of ROMs into various systems, rename, zip, etc. and then take that output and throw it through Igir to get only the ones for specific languages, or build ROM sets that are built for different handhelds, like the Miyoo Mini Plus or the RGB30. I'll probably eventually switch to only Igir, but right now this is working extremely well and it's saving me hours of cleanup time.

Nice job on this project. You should be rather proud of yourself!

**Comment by emmercm** [comment]
Thank you for your kind words @ndguardian! I'm fairly proud of the project, and I'm happy that it's had steady traction from the beginning. I've been trying to close feature gaps with other managers, hoping to make Igir a one-stop shop for most people (who don't mind or prefer a CLI experience).

Are there any specific features that Igir is missing when compared to RomVault, or more just ease of use or familiarity?

**Reply by ndguardian** [reply]
Hey, sorry just now saw you replied to this. Right now I take the hybrid approach simply due to familiarity. I think the more I use Igir though, I'll probably make it my main since it's so easy to automate. Feature-wise, I think the only thing Romvault really has (well, beyond DatVault, but that's it's own thing) that Igir doesn't is just the GUI, but obviously that wouldn't be in the spirit of Igir anyway.

**Reply by emmercm** [reply]
Yeah, I just don't see myself willing to take on a project like DatVault. It seems like it would need a lot of maintenance, and I don't think I have the time to devote. Never-ending projects like those tend to burn people out, like how Retool is recently unmaintained.
